util/packages.go

```diff
@@ -25,20 +27,26 @@ type Packages []Package
 // The list is stored in memory and on the second request directly served from memory.
 // This assumes changes to packages only happen on restart (unless development mode is enabled).
 // Caching the packages request many file reads every time this method is called.
-func GetPackages(packagesBasePaths []string) (Packages, error) {
+func GetPackages(ctx context.Context, packagesBasePaths []string) (Packages, error) {
+	span, ctx := apm.StartSpan(ctx, "GetPackages", "custom")
+	defer span.End()
```

Comment:
Given that the function just returns cached results if not already loaded, maybe it's not worth creating a span for this? It'll mostly be extremely fast, and when it's not then all the time will be attributed to GetPackagesFromFilesystem

Reply:
Good point. Currently the packages are cached in memory, and only reloaded on startup, but this is something that can change in the near future. There may be a need to "reload" packages, what could affect this method, or they may be cached in some shared external store.
Said that, we can add the span when this method does more things.